docs/token-guides/stmatic-superuser-functions.md

```diff
@@ -0,0 +1,53 @@
+# StMatic superuser functions
+
+<!--  -->
+
+## Superuser privileges, accounts, and roles
+
+StMatic token is an upgradable contract. The Lido for Polygon DAO can change the implementation with the successful DAO vote.
+
+StMatic can be stopped by the DAO vote. No operations changing StMatic balances can be performed on the stopped contract:
+1. `approve` and `transfer` calls revert;
+2. No mints or burns can be performed. Note that StMatic contract can mint StMatic only in two cases: 
+   user deposits (tokens are minted to the depositor's address) or 
+   fee distribution (where tokens are minted in accordance to fee calculations to the addresses set in the contract — namely the DAO treasury, the insurance fund and the Node Operator's reward addresses);
+3. Users can't submit their MATIC to the StMatic contract;
+4. No MATIC buffered in the StMatic contract can be sent to the Stake Manager;
+5. Staking withdrawals (once available) can't be performed.
+
+StMatic contract specifies two roles: **DEFAULT_ADMIN_ROLE** (address can pause the protocol by calling the `togglePause()` function) 
+and the **DAO_ROLE**. The DEFAULT_ADMIN_ROLE is assigned to the multi-sig contract [https://polygonscan.com/address/0xd65Fa54F8DF43064dfd8dDF223A446fc638800A9](https://polygonscan.com/address/0xd65Fa54F8DF43064dfd8dDF223A446fc638800A9). 
```

Comment:
This should be on Etherscan instead of Polygonscan

Reply:
will fix.

Reply:
done